docs/en/administration/fault_diagnosis_and_analysis.md

```diff
@@ -60,20 +60,19 @@ cat /var/log/syslog | grep 'juicefs' | grep '<ERROR>'
 
 ### Kubernetes CSI Driver
 
-Depending on the version of the JuiceFS CSI Driver you are using, there will be different ways to obtain logs. For details, please refer to [CSI Driver documentation](https://juicefs.com/docs/csi/troubleshooting).
+Depending on the version of the JuiceFS CSI Driver you are using, there will be different ways to retrieve logs. For details, please refer to [CSI Driver documentation](https://juicefs.com/docs/csi/troubleshooting).
 
 ### S3 Gateway
 
-The S3 gateway only supports running in the foreground, so client logs are output directly to the terminal. If you are deploying the S3 gateway in Kubernetes, you need to view the logs for the corresponding pod.
+The S3 gateway can only run in the foreground, so client logs are output directly to the terminal. If you deploys the S3 gateway in Kubernetes, you can get logs from the corresponding pods.
```

Comment:
```suggestion
The S3 gateway can only run in the foreground, so client logs are printed directly in the terminal. If the S3 gateway is deployed in Kubernetes, you can retrieve the corresponding pod logs.
```
Based on the Chinese document, if I understand correctly, the last sentence says that "if ...., we need to check the pod logs". But why we NEED to check the pod logs? Are the pod logs the prerequisites for other logs? Or it is just "get/retrieve the pod logs"? Sorry for the newbie question. :P

Reply:
> so client logs are printed directly in the terminal.

"printed to terminal" and "output to console" are both ok, they are used quite often.

and for the kubernetes part, I think it'll be fine with:

If deployed in Kubernetes, view the pod logs.

Reply:
@AIXjing It's NO NEED to check the pods logs. I guess the author of the Chinese doc was expressing "you need to check logs of the corresponding pods (rather than somewhere else) if you need to"